flink-autoscaler/src/main/java/org/apache/flink/autoscaler/tuning/MemoryTuning.java

```diff
@@ -277,6 +286,32 @@ private static MemorySize adjustNetworkMemory(
         return new MemorySize(memBudget.budget(maxNetworkMemory));
     }
 
+    /**
+     * Calculate how many network segment current vertex needs.
+     *
+     * @param currentVertexParallelism The parallelism of current vertex.
+     * @param otherVertexParallelism The parallelism of other vertex.
+     */
+    @VisibleForTesting
+    static int calculateNetworkSegmentNumber(
+            int currentVertexParallelism,
+            int otherVertexParallelism,
```

Comment:
```suggestion
            int upstreamVertexParallelism,
            int downstreamVertexParallelism,
```

Reply:
Can we change the logic to use these consistently? I think that makes reading the code a lot easier. Currently, the parameters change based on the ship strategy.

Reply:
IIUC, if we using the `upstream` and `downstream`, we still need to know which one is currentVertex, which one is otherVertex. For example, rebalance type consider the `otherVertexParallelism` as the channelCount.

Please correct me if my understanding is wrong or I miss anything, thank you

Reply:
"Other" just seems unspecific. How about this naming:
```suggestion
            int currentVertexParallelism,
            int connectedVertexParallelism,
```